core_lib/src/tool/basetool.cpp

```diff
@@ -148,6 +149,11 @@ void BaseTool::setBezier(const bool _bezier_state)
     properties.bezier_state = _bezier_state;
 }
 
+void BaseTool::setClosed(const bool _closed)
+{
+    properties.closedPolylinePath = _closed;
```

Comment:
```suggestion
void BaseTool::setClosed(const bool closed)
{
    properties.closedPolylinePath = closed;
```

There's no need for underscore here

Reply:
I had simply copied the code from right above 😛
https://github.com/pencil2d/pencil/blob/327ccbc6be4a8a6244f9a2d793ec545ac6b58fb9/core_lib/src/tool/basetool.cpp#L147-L150